OpenTokSamples/7.Multiparty-ConstraintLayout/README.md

```diff
@@ -2,15 +2,26 @@
 
 Note: Read the README.md file in the Project 1 folder before starting here.
 
-In the Project of the Sample #6 we built a simple multiparty app which uses basic Layouts in order to display the different video views of the participants.
+Sample #6 shows a simple multiparty app which uses basic layouts to display the different video
+views of the participants.
 
-However, when building a complex app, there is a better option in order to lay out the views. The recommended way is to use the relatively new `ViewGroup` called `ConstraintLayout`. Using this layout has several benefits, one of them is that you can use a single layout to specify all the view positions instead of having to nest several different layouts.
+However, when building a complex app, there is a better option in order to lay out the views:
+the relatively new `ViewGroup` called `ConstraintLayout`. Using this layout has several benefits.
+One  is that you can use a single layout to specify all the view positions instead of having to
+nest several different layouts.
+
+To use this sample application, install the ConstraintLayout for Android. In the Android Studio
```

Comment:
Yep, sounds good, we can point them to https://developer.android.com/training/constraint-layout/index.html#add-constraintlayout-to-your-project

Can you include it in this PR?